src/util/birthdaySystem/loop.ts

```diff
@@ -0,0 +1,57 @@
+import { Client, EmbedBuilder, Guild } from "discord.js";
+import { schedule } from "node-cron";
+import { config } from "../../config";
+import { birthdayDb } from "../../db";
+import "dotenv/config";
+import { addDefaultEmbedFooter } from "../misc/embeds";
+
+export async function BirthdayMessage(client: Client) {
+	schedule("0 * * * *", async () => {
+		if (!process.env.DEFAULT_GUILD)
+			throw new Error("This error shouldn't accure... Something is wrong with the env");
+
+		const targetHour = config.birthdayNotificationAt - 1;
+		const date = new Date();
+		const timezone = targetHour - date.getUTCHours();
+		const defaultGuild = await client.guilds.fetch(process.env.DEFAULT_GUILD);
+		const possibleUsers =
+			date.getUTCHours() == 0
+				? getMembersWithNoTimeZone(defaultGuild) || []
+				: getMembersWithTimeZone(
+						`UTC${timezone > 0 ? "+" + timezone : timezone < 0 ? timezone : ""}`,
+						defaultGuild
+				  ) || [];
+		const adjustedDate = new Date(Date.now() + timezone * 60 * 60 * 1000);
+		const adjustedMonth = adjustedDate.getUTCMonth() + 1;
+		const adjustedDay = adjustedDate.getUTCDate();
+		for (const user of possibleUsers) {
+			const { day, month } = birthdayDb.get(user.id) ?? {};
+			if (!day || !month) continue;
+
+			if (
+				(adjustedMonth === month && adjustedDay === day) ||
+				(month === 1 && day === 29 && adjustedMonth === 2 && adjustedDay === 1)
```

Comment:
0-11 vs 1-12
```suggestion
				(month === 2 && day === 29 && adjustedMonth === 3 && adjustedDay === 1)
```

Reply:
Isnt this already done with `const adjustedMonth = adjustedDate.getUTCMonth() + 1;` and `const targetHour = config.birthdayNotificationAt - 1;`?